src/pipeline.rs

```diff
@@ -232,27 +231,19 @@ impl SpecializedMeshPipeline for OutlinePipeline {
             self.mesh_pipeline.view_layout_multisampled.clone()
         }];
         let mut buffer_attrs = vec![Mesh::ATTRIBUTE_POSITION.at_shader_location(0)];
-        let mut vertex_defs = vec![
```

Comment:
I suspect these are also related to the shader imports... What I was running into errors like this for the both of them

```
    src/outline.wgsl: multiple inconsistent shader def values: 'MAX_DIRECTIONAL_LIGHTS'
```

when I removed these, the errors went away. I suspect that they just are already in scope now and since the value of these comes from bevy::pbr that it might be alright to just use what is already there?